src/core/installAddon.js

```diff
@@ -407,6 +419,18 @@ export class WithInstallHelpers extends React.Component<WithInstallHelpersIntern
           category: getAddonEventCategory(type, INSTALL_ACTION),
           label: guid,
         });
+
+        const conversionInfo = _getConversionInfo();
+        if (conversionInfo) {
+          const { addonId, data } = conversionInfo;
+
+          if (addonId === id && data) {
+            _sendSponsoredEventBeacon({ data, type: 'conversion' });
```

Comment:
We could probably move the whole logic inside a specialized function and only inject 1 function instead of 3.

Something like `_trackConversion()` or something

Reply:
Fixed!